sre-recipes/recipes/recipe-1/kubernetes-manifests/frontend.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
```

Comment:
How different is this manifest from the standard frontend manifest?

Reply:
It's actually identical except for the environment variables. I think setting the env var using `kubectl set env` is a good solution @tishen25 if we can adapt this into the CLI that would probably be best

Reply:
Yup! It has been added to [this PR](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/pull/422).